src/modules/securitybaseline/src/lib/SecurityBaseline.c

```diff
@@ -2937,7 +2937,7 @@ static int InitEnsureDenyGroupsConfigured(char* value)
 
 static int InitEnsureSshHostbasedAuthenticationIsDisabled(char* value)
 {
-    return InitializeSshAuditCheck(g_initEnsurePermissionsOnEtcSshSshdConfigObject, value, SecurityBaselineGetLog());
+    return InitializeSshAuditCheck(g_initEnsureSshHostbasedAuthenticationIsDisabledObject, value, SecurityBaselineGetLog());

```

Comment:
Host based is two words, capitalize the `b` -> `(g_initEnsureSshHostBasedAuthenticationIsDisabledObject`?

Reply:
Too late to leave feedback on this. Note same name is everywhere, including in the policy, and was added ~ 1 year ago. The original OpenSSH setting is named "Host-based Authentication", why "host" is lowercase in the name.